pkg/sql/testdata/pg_catalog_test_expected_diffs.json

```diff
@@ -7,43 +7,17 @@
       "expectedDataType": "regproc"
     }
   },
-  "pg_attrdef": {
-    "adbin": {
-      "oid": 25,
-      "dataType": "text",
-      "expectedOid": 194,
-      "expectedDataType": "pg_node_tree"
-    }
-  },
   "pg_attribute": {
-    "attacl": {
-      "oid": 1009,
-      "dataType": "_text",
-      "expectedOid": 1034,
-      "expectedDataType": "_aclitem"
-    },
     "attmissingval": null
   },
   "pg_class": {
-    "relacl": {
-      "oid": 1009,
-      "dataType": "_text",
-      "expectedOid": 1034,
-      "expectedDataType": "_aclitem"
-    },
     "relfrozenxid": {
       "oid": 20,
       "dataType": "int8",
-      "expectedOid": 28,
-      "expectedDataType": "xid"
+      "expectedOid": 26,
+      "expectedDataType": "OID"
     },
-    "relminmxid": null,
-    "relpartbound": {
-      "oid": 25,
-      "dataType": "text",
-      "expectedOid": 194,
-      "expectedDataType": "pg_node_tree"
-    }
+    "relminmxid": null
```

Comment:
so now the next step should be that we generate missing columns like this. (it can happen in a later PR)

does the code generation tool automatically to that?

Reply:
Yes, the code generation tool will automatically do that with this change, this is the PR #64035